tests/modules/testrdb.c

```diff
@@ -98,6 +98,8 @@ void testrdb_aux_save(ValkeyModuleIO *rdb, int when) {
     if (!(conf_aux_count & CONF_AUX_OPTION_BEFORE_KEYSPACE)) assert(when == VALKEYMODULE_AUX_AFTER_RDB);
     if (!(conf_aux_count & CONF_AUX_OPTION_AFTER_KEYSPACE)) assert(when == VALKEYMODULE_AUX_BEFORE_RDB);
     assert(conf_aux_count!=CONF_AUX_OPTION_NO_AUX);
+    ValkeyModuleCtx *ctx = ValkeyModule_GetContextFromIO(rdb);
+    VALKEYMODULE_NOT_USED(ctx);
```

Comment:
We could add a code comment explaining why we do this, that it has a side effect. Otherwise it looks like redundant code.